page_revisions.go

```diff
@@ -2,7 +2,13 @@ package mediawiki
 
 import "time"
 
-const revisionsURL = "/w/api.php?action=query&format=json&prop=revisions&rvlimit=%d&rvdir=%s&formatversion=2&titles=%s"
+const revisionsURL = "/w/api.php?"
+
+// PageRevisionsOptions additional optional parameters for PageRevisions method
+type PageRevisionsOptions struct {
+	Order RevisionOrdering
+	Props []string
```

Comment:
Isn't this RevisionProps as well ?

Reply:
Well, since this Options refer to the `Revision` entity, it's redundant to have the `Revision` itself in the name of the field.

In the case of `PageDataOptions`, the entity we refer is the page itself, that may or may not contain revisions information. That's why in that case it is called `RevisionProps`.